src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/AssessmentEpisodeEntity.kt

```diff
@@ -49,7 +49,7 @@ class AssessmentEpisodeEntity(
   val userId: String? = null,
 
   @Column(name = "CREATED_DATE")
-  val createdDate: LocalDateTime? = null,
+  val createdDate: LocalDateTime = LocalDateTime.now(),
 
```

Comment:
createdDate is not nullable

CREATE TABLE IF NOT EXISTS assessed_episode
(
    episode_id            SERIAL PRIMARY KEY,
    episode_uuid          UUID UNIQUE,
    user_id               TEXT        NOT NULL,
    assessment_schema_code VARCHAR(50) NULL,
    oasys_set_pk          BIGINT      NULL,
    created_date          TIMESTAMP   NOT NULL,
    end_date              TIMESTAMP,
    change_reason         TEXT        NOT NULL,
    assessment_uuid       UUID        NOT NULL,
    answers               JSONB,
    FOREIGN KEY (assessment_uuid)
        REFERENCES assessment (assessment_uuid)
);